charts/kube-prometheus-stack/Chart.yaml

```
@@ -22,7 +22,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 38.0.1
+version: 38.1.0
```

as grafana only raised the bugfix version i would go with:
```suggestion
version: 38.0.2
```